README.md

```diff
@@ -4,54 +4,48 @@ Shell script alias manager.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
 
 ### Prerequisites
 
-What things you need to install the software and how to install them
-
-```
-Give examples
-```
+Before starting with Ali you must have installed Go.
```

Comment:
What about `Ali requirements for development are Go and a unix based system.` instead?

Reply:
More much better.